src/modules/dashboard/components/ColonyActions/ColonyActions.tsx

```diff
@@ -264,22 +274,32 @@ const ColonyActions = ({
     <div className={styles.main}>
       {sortedActionsData?.length ? (
         <>
-          <Form
-            initialValues={{ filter: SortOptions.NEWEST }}
-            onSubmit={() => undefined}
-          >
-            <div className={styles.filter}>
-              <Select
-                appearance={{ alignOptions: 'left', theme: 'alt' }}
-                elementOnly
-                label={MSG.labelFilter}
-                name="filter"
-                options={SortSelectOptions}
-                onChange={setActionsSortOption}
-                placeholder={MSG.placeholderFilter}
-              />
+          <div className={styles.bar}>
+            <div className={styles.title}>
+              <FormattedMessage {...MSG.actionsTitle} />
             </div>
-          </Form>
+
+            <Form
+              initialValues={{ filter: SortOptions.NEWEST }}
+              onSubmit={() => undefined}
+            >
+              <div className={styles.filter}>
+                <Select
+                  appearance={{ alignOptions: 'left', theme: 'alt' }}
+                  elementOnly
+                  label={MSG.labelFilter}
+                  name="filter"
+                  options={SortSelectOptions}
+                  onChange={setActionsSortOption}
+                  placeholder={MSG.placeholderFilter}
+                />
+              </div>
+            </Form>
+
+            <Link className={styles.link} to={`/colony/${colonyName}/events`}>
+              <FormattedMessage {...MSG.transactionsLogLink} />
+            </Link>
```

Comment:
I think it is good to have the direct route still. Purely because otherwise it will lead to a lot of broken links (bookmarks, history).

Reply:
I'd argue that nobody in their right mind bookmarked the events page.

Than again, I'm good more or less either way, as long as we have some input from product.

So this can stay as it is for now. :+1: